src/phpDebug.ts

```diff
@@ -325,6 +330,30 @@ class PhpDebugSession extends vscode.DebugSession {
         }
         this.sendResponse(response)
     }
+    /**
+     * Opens the urls specified in configuration, either with start or stop query string
+     * @param {boolean} start
+     */
+    private _openUrl(start: boolean) {
+        if (!this._args.openUrl) {
+            return
+        }
+        const myurl = new url.URL(this._args.openUrl)
+        if (start) {
+            myurl.searchParams.append('XDEBUG_SESSION_START', 'vscode')
```

Comment:
I think this should be set, not append, since append would duplicate the parameter if it’s already set

Reply:
I would rather keep the append, as a the stop command would be confusing for the xdebug anyway

Reply:
I don't understand what this has to do with the stop command - what I mean is that `append` would potentially add `XDEBUG_SESSION_START` twice or more times, like `?XDEBUG_SESSION_START=vscode&XDEBUG_SESSION_START=vscode`, while `set()` would just override it so `XDEBUG_SESSION_START` appears exactly once

Reply:
but what would happen when the stop command should be sent? XDEBUG_SESSION_START=vscode&XDEBUG_SESSION_STOP=vscode (with set or append) as the param is diferrent, so it would be easier to spot such errors.